static/resources/harness.js

```diff
@@ -338,7 +338,11 @@
     css.rel = 'stylesheet';
     css.type = 'text/css';
     css.href = '/resources/style.css';
-    document.head.appendChild(css);
+    try {
+      document.head.appendChild(css);
+    } catch(e) {
+      // If the CSS fails to load, oh well
```

Comment:
In which browser does `document.head.appendChild(css)` throw an exception? Any exception shouldn't be due to the CSS failing to load, which happens asynchronously, but more likely because `document.head` doesn't exist. I'm guessing this happened on IE<9, based on https://developer.mozilla.org/en-US/docs/Web/API/Document/head?

Reply:
It was actually early versions of Chrome (like Chrome 4), which led me to believe that it may occur in other browsers as well (like IE).  I tried with `document.getElementsByTagName('head')[0]` as well, but had no luck there.

Reply:
I see. Catching and ignoring the error is fine. Only the comment needs updating, something like "If inserting the link element fails" would be more accurate about where the exception is coming from. (The consequence will be the CSS won't load, though.)